enterprise/frontend/src/metabase-enterprise/auth/index.js

```diff
@@ -181,6 +181,12 @@ PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections => ({
         required: true,
         widget: SecretKeyWidget,
         getHidden: settings => !settings["jwt-enabled"],
+        props: {
+          confirmation: {
+            header: t`Regenerate JWT signing key?`,
+            dialog: t`This will cause existing tokens to stop working until the identity provider is updated with the new key.`,
+          },
+        },
```

Comment:
Can we a do a similar thing like this for embedding key regeneration, so that `SecretKeyWidget` is being used in the same way for both use cases?
```typescript
props: {
  confirmation: {
    header: t`Regenerate embedding key?`,
    dialog: t`This will cause existing embeds to stop working until they are updated with the new key.`,
  },
},
```

- Embedding component is setup in this file: `/metabase/frontend/src/metabase/admin/settings/selectors.js` (lines 512 - 518)

Reply:
Done